shared-kernel-errors/src/main/java/org/dcsa/skernel/errors/infrastructure/ConcreteRequestErrorMessageExceptionHandler.java

```diff
@@ -22,6 +22,7 @@ public class ConcreteRequestErrorMessageExceptionHandler extends BaseExceptionHa
   @ExceptionHandler(ConcreteRequestErrorMessageException.class)
   public ResponseEntity<RequestFailureTO> handleConcreteRequestErrorMessageException(
     HttpServletRequest httpServletRequest, ConcreteRequestErrorMessageException e) {
+    log.debug("Exception {}: {}", e.getClass().getName(), e.getMessage());
```

Comment:
I would prefer if we did not have a debug logging here.  During IT tests, this will cause a lot of noise for 400 and 404 cases.